### ISSUE TITLE
Prettier precommit runs on merge commits

### ISSUE THREAD

**Issue by jnaviask** [hicommonwealth/commonwealth#3943]
## Description
I tried to fix merge conflicts locally, and the prettier precommit hook started running on every file in the merge, which took an extremely long time to complete.

## Acceptance Criteria
- Prettier precommit hook should not run when committing merges.

**Comment by timolegros** [comment]
I believe this is already the case. I just tested this by resolving a conflict with something that would have triggered the hook but it did not run when I committed the merge/conflict resolution (then made another commit and the code used to resolve the conflict was correctly formatted by prettier). If this is still happening for you I will need some info on how to replicate.

**Comment by jnaviask** [comment]
@timolegros try the following:
1. Start a branch off an old commit.
2. Using the CLI, merge master into the branch.
3. See if the linter runs on the merged files.

If the linter doesn't touch the merged files, then we can close this ticket. Otherwise, the problem is outstanding.

**Comment by timolegros** [comment]
@jnaviask Still unable to reproduce. I did as follows using the Git CLI (used WebStorm interface in my first attempt before):
1. Branched off an old master commit
2. Made a change that I knew would create a merge conflict
3. Started merging master
4. Resolved the conflict in a way that prettier would reformat
5. Finalized the merge and committed the changes

Prettier did not run and the file remained unformatted following the merge. I am curious though because no changes were made to the hook before (since creation) or after your issue was created so it seems like this may be an environment-specific issue.
